.npmignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
```

Comment:
When updating the Dispensing module to use yarn to publish instead of npm I discovered I had to add this file because of this issue:

https://github.com/yarnpkg/yarn/issues/754

Please feel free to rework if there's a better way to do this... I noticed that some of the other ESM projects are using different commands when publishing.

Reply:
IIRC, if we migrate to a newer version of Yarn it's a non-issue. The monorepos are using all using [Lerna](https://lerna.js.org/) (albeit an older version) to publish.